test/ja_serializer/builder/relationship_test.exs

```diff
@@ -22,7 +22,13 @@ defmodule JaSerializer.Builder.RelationshipTest do
 
   defmodule FooSerializer do
     use JaSerializer
-    has_many :bars, type: "bar"
+    has_many :bars,
+             type: "bar",
+             relationship_links: [
```

Comment:
I originally had `links` and flip-flopped on what to name the key.  I am totally fine with just "links"